windows/jitsi-meet/micro.js

```diff
@@ -0,0 +1,124 @@
+const {ipcRenderer} = require('electron');
+
+/**
+ * The RTCPeerConnection to main window.
+ */
+let peerConnectionMicro;
+
+/**
+ * The remote video from main window.
+ */
+let largeVideo = document.createElement('video');
```

Comment:
Nope, it is for convenience sake for development for now. In the end, all the element will be defined in html.